app/src/main/java/com/dessalines/thumbkey/db/AppDb.kt

```diff
@@ -209,8 +215,16 @@ val MIGRATION_6_7 = object : Migration(6, 7) {
     }
 }
 
+val MIGRATION_7_8 = object : Migration(7, 8) {
+    override fun migrate(database: SupportSQLiteDatabase) {
+        database.execSQL(
+            "alter table AppSettings add column hide_symbols BOOLEAN NOT NULL default $DEFAULT_HIDE_SYMBOLS",
```

Comment:
Whew, good thing I tested this, because it breaks existing installations. This needed to be an INTEGER, not a boolean. I'm fixing it now.